libyul/AsmParser.h

```diff
@@ -114,6 +114,12 @@ class Parser: public langutil::ParserBase
 		langutil::SourceLocation const& _commentLocation
 	);
 
+	std::optional<std::pair<std::string_view, std::optional<int>>>
+	parseASTIDComment(
```

Comment:
Why not this? It's not that long.
```suggestion
	std::optional<std::pair<std::string_view, std::optional<int>>> parseASTIDComment(
```